reccmp/tools/stackcmp.py

```diff
@@ -277,66 +284,43 @@ def virtual_address(value) -> int:
     parser.add_argument(
         "--version", action="version", version=f"%(prog)s {reccmp.VERSION}"
     )
-    parser.add_argument(
-        "original", metavar="original-binary", help="The original binary"
-    )
-    parser.add_argument(
-        "recompiled", metavar="recompiled-binary", help="The recompiled binary"
-    )
-    parser.add_argument(
-        "pdb", metavar="recompiled-pdb", help="The PDB of the recompiled binary"
-    )
-    parser.add_argument(
-        "decomp_dir", metavar="decomp-dir", help="The decompiled source tree"
-    )
+    argparse_add_built_project_target_args(parser)
 
     parser.add_argument(
         "address",
         metavar="<offset>",
         type=virtual_address,
         help="The original file's offset of the function to be analyzed",
     )
-
-    parser.set_defaults(loglevel=logging.INFO)
-    parser.add_argument(
-        "--debug",
-        action="store_const",
-        const=logging.DEBUG,
-        dest="loglevel",
-        help="Print script debug information",
-    )
+    argparse_add_logging_args(parser)
 
     args = parser.parse_args()
 
-    if not os.path.isfile(args.original):
-        parser.error(f"Original binary {args.original} does not exist")
-
-    if not os.path.isfile(args.recompiled):
-        parser.error(f"Recompiled binary {args.recompiled} does not exist")
-
-    if not os.path.isfile(args.pdb):
-        parser.error(f"Symbols PDB {args.pdb} does not exist")
-
-    if not os.path.isdir(args.decomp_dir):
-        parser.error(f"Source directory {args.decomp_dir} does not exist")
+    argparse_parse_logging(args=args)
+    if args.loglevel != logging.DEBUG:
+        # Mute logger events from compare engine
+        logging.getLogger("isledecomp.compare.core").setLevel(logging.CRITICAL)
```

Comment:
The `--verbose` option has different meaning in `stackcmp.py` and `reccmp.py`.
I think we should find out:why we have to disable the logging event from the isledecomp module in the first place.
Perhaps it should return a list of errors, throw an exception, or log with a lower priority (debug?).
If a module is too spammy by default, then logging becomes less useful.

Reply:
* I'll create a ticket for the future so this doesn't block the merge
* I remember that I added some additional mutes to one of them because it annoyed me while working on BETA10